src/test/java/reposense/parser/CsvParserTest.java

```diff
@@ -229,6 +231,12 @@ public void authorConfig_invalidLocation_success() throws IOException {
         Assert.assertEquals(3, config.getAuthorList().size());
     }
 
+    @Test (expected = IOException.class)
+    public void authorConfig_invalidHeaderSize_throwsIoException() throws IOException {
+        AuthorConfigCsvParser authorConfigCsvParser = new AuthorConfigCsvParser(AUTHOR_CONFIG_INVALID_HEADER_SIZE);
+        authorConfigCsvParser.parse();
+    }
```

Comment:
Do the same set of test cases for group and repo config  as well

Reply:
Done